plugins/out_oracle_log_analytics/oci_logan.c

```diff
@@ -278,6 +278,13 @@ static int build_headers(struct flb_http_client *c, struct flb_oci_logan *ctx,
         goto error_label;
     }
 
+    tmp_sds = NULL;
```

Comment:
this initialization is not needed since the next line will set the value

Reply:
sorry for mistakes. thank you for the changes.